src/test/java/edu/ucla/library/iiif/auth/utils/TestUtils.java

```diff
@@ -44,8 +51,8 @@ private TestUtils() {
      *         is a {@link CookieNames#SINAI_IV} cookie
      * @throws Exception If there is an issue generating the cookies
      * @see <a href=
-     *      "https://github.com/UCLALibrary/sinaimanuscripts/blob/44cbbd9bf508c32b742f1617205a679edf77603e/app/controllers/application_controller.rb#L98-L103">How
-     *      the Sinai application encodes cookies</a>
+     *      "https://github.com/UCLALibrary/sinaimanuscripts/blob/main/app/controllers/application_controller.rb">How
```

Comment:
Per our side channel discussion @ksclarke : 
```suggestion
     *      "https://github.com/UCLALibrary/sinaimanuscripts/blob/v2.15.7/app/controllers/application_controller.rb">How
```

Reply:
Or comment in code instead is fine w/ me if you prefer

Reply:
See ongoing discussion... I really think this should be a comment in the code (to the tagged version, yes) instead of a see tag which I see as more linking to real documentation.